store/src/java/com/zimbra/util/ExternalVolumeInfoHandler.java

```diff
@@ -192,4 +192,35 @@ public void modifyServerProperties(VolumeInfo volInfo) throws ServiceException,
             throw e;
         }
     }
+
+    /**
+     * Validate global bucket ID by reading LDAP atrribute
+     * @param globalS3BucketId
+     * @returns true if "global bucket ID" is valid
+     * @throws ServiceException, JSONException
+     */
+    public Boolean validateGlbBucketID(String globalS3BucketId) throws ServiceException, JSONException {
```

Comment:
could rename method to `validateGlobalBucketID` from `validateGlbBucketID`

Reply:
Done